opencti-platform/opencti-front/src/private/components/common/stix_domain_objects/StixDomainObjectAttackPatternsKillChain.jsx

```diff
@@ -272,7 +272,7 @@ class StixDomainObjectAttackPatternsKillChainComponent extends Component {
               entityLink={entityLink}
               paginationOptions={paginationOptions}
               onDelete={this.props.relay.refetch.bind(this)}
-              searchTerm={searchTerm}
+              searchTerm={searchTerm ?? ''}
```

Comment:
Shouldn't this be handled by the view storage of the parent component's StixDomainObjectAttackPatterns?
Or I'm missing something.

Reply:
It's not handled by view storage, but can be handled by the parent right!

Reply:
fixed